src/main/resources/liquibase.properties

```diff
@@ -0,0 +1,4 @@
+url=jdbc:mysql://localhost:3306/bookstore
+username=root
+password=1234
+changelog-file=db/changelog/db.changelog-master.yaml
```

Comment:
It's not safe to add your credentials to remote repo

Reply:
Agree